plugin/Form/FormAddressCreator.php

```diff
@@ -68,7 +68,7 @@ public function __construct(
 	 * @throws NotFoundException
 	 */
 	public function createOrCompleteAddress(
-		FormData $pFormData, bool $mergeExisting = false, string $contactType = ''): int
+		FormData $pFormData, bool $mergeExisting = false, string $contactType = '', int $estateId = null): int
```

Comment:
Could you please add PHPDoc for these arguments: $contactType and $estateId for complete documentation? :)